typescript/infra/config/environments/mainnet3/aw-validators/rc.json

```diff
@@ -141,13 +141,6 @@
       "0xcaa9c6e6efa35e4a8b47565f3ce98845fa638bf3"
     ]
   },
-  "neutron": {
-    "validators": [
-      "0x307a8fe091b8273c7ce3d277b161b4a2167279b1",
-      "0xb825c1bd020cb068f477b320f591b32e26814b5b",
-      "0x0a5b31090d4c3c207b9ea6708f938e328f895fce"
-    ]
-  },
```

Comment:
I wonder why this was moved to the end?

Reply:
honestly unsure for both questions - this file is auto generated. I'm happy to undo the removals but it'll likely get removed for a similar reason on another rerun